get-bridge/homebrew-tap
# Pinning Truss Local v0.3.0

With the upcoming merge of https://github.com/get-bridge/truss-local/pull/37 supporting Netskope in Truss Local, I'd like to give people the ability to revert back to v0.3.0 in case things don't work as expected. This PR simply copies and renames the existing Truss Local formula under the name `truss-local@0.3.0`.

Since v0.3.0 is also the latest version, I also pinned v0.2.2 for testing/verification purposes.

[Goreleaser](https://goreleaser.com/customization/homebrew/#versioned-formulas) supports doing this automatically in its Pro version. :(

Test Plan:
- `docker run --rm -it homebrew/brew bash`
  - Install our Tap
    - `export HOMEBREW_GITHUB_API_TOKEN=...` (from [here](https://github.com/settings/tokens/new?scopes=repo&description=Homebrew%20for%20Bridge%20Tools))
    - `brew tap get-bridge/tap https://github.com/get-bridge/homebrew-tap`
    - `cd /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/get-bridge/homebrew-tap && git merge pin-truss-local-0.3.0`
  - Search for versions
    - `brew search truss-local`
  - Install v0.2.2
    - `brew install truss-local@0.2.2`
    - `truss-local --version`
      Should reveal `truss-local version v0.2.2`.
  - Install v0.3.0
    - `brew install truss-local@0.3.0`
    - `truss-local --version`
      Should reveal `truss-local version v0.3.0`.
